zorg/buildbot/builders/annotated/libc-linux.py

```diff
@@ -111,7 +111,11 @@ def main(argv):
             cmake_args.append('-DCMAKE_LINKER=/usr/bin/ld.lld')
             cmake_args.append('-DLLVM_LIBC_MPFR_INSTALL_PATH={}/gmp+mpfr/'.format(os.getenv('HOME')))
 
-        run_command(['cmake', os.path.join(source_dir, 'llvm')] + cmake_args)
+        if bootstrap_build:
```

Comment:
maybe we want this for fullbuild_build, too?

Reply:
if scudo was built in fullbuild_build with runtimes as root then I think it's fine?